core/arch/arch_exports.h

```diff
@@ -1663,10 +1663,10 @@ d_r_decode_init(void);
 #    define STUB_COARSE_DIRECT_SIZE(flags) \
         (FRAG_IS_32(flags) ? STUB_COARSE_DIRECT_SIZE32 : STUB_COARSE_DIRECT_SIZE64)
 
-/* Writes nops into the address range.
- * XXX i#3828: Better to use the newer multi-byte nops.
- */
-#    define SET_TO_NOPS(isa_mode, addr, size) memset(addr, 0x90, size)
+/* Writes nops into the address range. */
+bool
```

Comment:
We can pull out this declaration from the #ifdef X86 #elif defined(AARCHXX), saving one line.